WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -38,20 +38,28 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v19.0-whats-new"
+msgctxt "v19.1-whats-new"
 msgid ""
-"What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.\n"
+"Only gave the app limited access to your media library? No big deal, but we added a prompt to the media picker in case you want to change your settings and allow more access. The prompt even comes in dynamic type for easy reading.\n"
 "\n"
-"If you’re using a free plan, video uploads are now limited to five minutes per file. Don’t worry, that won’t affect any videos you’ve already uploaded to your site.\n"
+"We also made voiceover and dynamic type more accurate on Activity Log and Schedule Post calendars. We’re happy to report that this improves the user experience for folks who need better visual accessibility.\n"
```

Comment:
I wonder if "dynamic type" is a term that end users will understand… but also I have no idea about what better alternative wording we could use to describe that, so 🤷

Reply:
Yeah, good point... 🤔 

iOS explicitly uses it, even though it's in the "Text Size" screen, so it's not "in the user face"

<img alt="IMG_B587EB3F0BE3-1" src="https://user-images.githubusercontent.com/1218433/151402909-fd1cbb5b-5e2a-42d3-b9e9-906f4cd4a3a2.jpeg" width=300/>

Maybe it's that kind of thing only someone that uses accessibility features knows about?

Reply:
Yeah. I know that's the official term, but likely one that is completely foreign to most iPhone users who never used any accessibility settings.

I thought about mentioning accessibility is our wording there, to provide context for the "dynamic type" term. Which, now that I look back at the line I commented on, is already the case after all.
But it happens that the line I intended to comment on initially was [the one just above](https://github.com/wordpress-mobile/WordPress-iOS/pull/17831/files#diff-10a2b246761a7cf613af55011392f25688ed469b14258c70d27584e9f85a727fR43), instead of this one… and that one not mentioning the context of accessibility is what made me wonder this in the first place.

> The prompt even comes in dynamic type for easy reading.

Maybe given the next line reuses that "dynamic type" term but also puts it in the context of accessibility, that is enough after all.
And as I said, it was just a passing thought and I don't have any inspiration for any better suggestion anyway, so let's just keep it that way.

PS: Interestingly, I couldn't find any reference of "Dynamic Type" and (thus how Apple would translate this term in various locales) in [this nice site](https://applelocalization.com) that I recently discovered via [this tweet](https://twitter.com/k_katsumi/status/1486279448345575428)… 🙃

Reply:
Ohhh btw @mokagio ⚠️  I think the official spelling for `VoiceOver` is using titlecase, not lowercase like here?

I'm hoping Apple won't be so petty that they'd reject the build at submission time for us using `voiceover` instead of `VoiceOver` in our release notes… but we all know how petty they can be sometimes so we never know… maybe worth fixing just in case, and before it starts being translated all lowercase in all locales?